### DISCUSSION TITLE
We need to add a copy button in the ios app

### DISCUSSION THREAD

**Discussion by alialhammad1992** [community/community#29031] [Mobile]
![image](https://user-images.githubusercontent.com/104555394/183926335-88021f56-1a9c-4ec0-8360-0e5585a283b8.jpeg)

Please add it in the application

**Comment by haldun** [comment]
Hi @alialhammad1992, thanks for the feedback. I am passing this to the team for prioritization.

**Reply by alialhammad1992** [reply]
And when will the team start?
